docs/ADRs/012-tad-tool.md

```diff
@@ -0,0 +1,50 @@
+# ADR-0012 Tool for Transparency of Algorithmic Decision making
+
+## Context
+
+We are considering tooling for organisations to get more grip on their algorithms. Tooling for, for instance bias and fairness tests, and assessments (like [IAMA](https://www.rijksoverheid.nl/documenten/rapporten/2021/02/25/impact-assessment-mensenrechten-en-algoritmes)).
+
+Transparency, we think, can be fostered by sharing reports from such a tool in a standardized way.
+
+There are several existing open source
+[tools](https://minbzk.github.io/ai-validation/Projects/TAD/Existing%20Tools/Comparison/tools/) which
+we [have assessed](https://minbzk.github.io/ai-validation/Projects/TAD/Existing%20Tools/Comparison/tools_comparison/).
```

Comment:
```suggestion
[tools](../Projects/TAD/Existing%20Tools/Comparison/tools/) which
we [have assessed](../Projects/TAD/Existing%20Tools/Comparison/tools_comparison/).
```

We should really fix that url to have a `-` instead of a space though